tests/unit/core/test_tracking.js

```diff
@@ -279,6 +291,81 @@ describe(__filename, () => {
         getAddonEventCategory(ADDON_TYPE_STATIC_THEME, INSTALL_ACTION),
       ).toEqual(INSTALL_THEME_CATEGORY);
     });
+
+    it('returns the expected category when type is extension and installAction is cancelled', () => {
+      expect(
+        getAddonEventCategory(ADDON_TYPE_EXTENSION, INSTALL_CANCELLED_ACTION),
+      ).toEqual(INSTALL_EXTENSION_CANCELLED_CATEGORY);
+    });
+
+    it('returns the expected category when type is lightweight theme and installAction is cancelled', () => {
+      expect(
```

Comment:
I don't see this option to cancel with lightweight theme, do you?

Reply:
I see what you mean but for completeness, we should test that, no?

Reply:
yeh I see .. yep i think that makes sense : )